### PR TITLE
Discover mention

### PR DESCRIPTION
**Pull request by mkondratek**

### CODE REVIEW

File: blog.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ which be easily opened from the plugin's toolbar.
 It supports both syntax highlight and completion of local branch names.
 The graph is automatically refreshed when `.git/machete` is modified.
 
+Bear in mind that there is no need to edit the machete file manually to set up your branch layout tree. 
+Feel free to use `Discover` functionality that will find it out for you automatically.
+It can be found within the empty graph table and via `Find Action...`.
```
Comment (PawelLipski) [inline blog.md]: I don't like the use of `graph table` here, this might be a bit too technical/internal/unclear for the reader :/

Comment (PawelLipski) [reply blog.md]: `Find Action... (Ctrl+Shift+A)`

Comment (mkondratek) [reply blog.md]: I would not rely on the shortcuts since they differ between OSes & personal settings

**Review by PawelLipski — APPROVED** [review]

**Review by PawelLipski — APPROVED** [review]
No reason to keep it as a separate PR anymore, pls merge into `blog` branch (so that only PR #1 remains)

**Pull request merged**